pygen/pygen_src/isa/riscv_cov_instr.py

```diff
@@ -19,8 +19,12 @@
 import logging
 from enum import Enum, IntEnum, auto
 from bitstring import BitArray
-from pygen_src.target.rv32i import riscv_core_setting as rcs
 from pygen_src.riscv_instr_pkg import *
+from pygen_src.riscv_instr_gen_config import cfg
+if cfg.argv.target == "rv32i":
+    from pygen_src.target.rv32i import riscv_core_setting as rcs
+if cfg.argv.target == "rv32imc":
```

Comment:
Hi @aneels3 

Are you going to replace these `if target==X` with the information coming from the command line arguments in this PR? Or we should expect it in a separate PR? Here as well as the nested if statements part (I believe we discussed it during our last meeting).

Reply:
Hi @hodjat91 
Sorry for the delay.
I am thinking to create a separate PR for this issue along with a tabular form of config_gen fields.

Reply:
Hi @hodjat91 
I have fixed this 
Please review it once.